src/components/PageHelp.tsx

```diff
@@ -87,6 +87,32 @@ const Help = () => (
             name consists from the repository name plus the package, e.g.
             @ksh/latest/ksh.
         </p>
+        <h3>Via GraphQL</h3>
+        <p>
+            You can construct GraphQL queries and execute them on the dashboard's 
+            backend GraphQL server. The GraphQL backend is usually located at:
+        </p>
+        <p className="padding-left-20">
```

Comment:
What is this class, `padding-left-20`? It doesn't appear to be a PatternFly class and we don't define it anywhere.

Edit: Ah, I see it was used in the page before. We should get rid of it.

Reply:
Should I keep this class for now?

Reply:
I think yes, in different PR.